src/index.js

```diff
@@ -335,17 +342,40 @@ class ServerlessIOpipePlugin {
       throw new Error(err);
     }
   }
+  getConfigFromCosmi() {
+    let { token } = this.getOptions();
+    const { config: cosmi = {} } =
+      cosmiconfig('iopipe', {
+        cache: false,
+        sync: true,
+        rcExtensions: true
+      }).load(process.cwd()) || {};
+    const requireLines = _.chain(cosmi.plugins)
+      .map(p => `require('${p}');\n`)
+      .join()
+      .defaultTo('')
+      .value();
+    const inlineConfigObject = _.pickBy(_.assign({}, cosmi, { token }));
+    const inlineConfig = _.isEmpty(inlineConfigObject)
+      ? ''
+      : JSON.stringify(inlineConfigObject);
+    return {
+      requireLines,
+      inlineConfig
+    };
+  }
   createFile() {
     const debug = createDebugger('createFile');
     debug('Creating file');
-    let { token } = this.getOptions();
-    const agentConfig = token ? `{token: '${token}'}` : '';
-    const iopipeInclude = `const iopipe = require('iopipe')(${agentConfig});`;
+    const { inlineConfig, requireLines } = this.getConfigFromCosmi();
+    const iopipeInclude = `${requireLines}const iopipe = require('${this.getInstalledPackageName()}')(\n${inlineConfig}\n);`;
     const funcContents = _.chain(this.funcs)
       .map(outputHandlerCode)
       .join('\n')
       .value();
-    const contents = `${iopipeInclude}\n\n${funcContents}`;
+    const contents = prettier.format(`${iopipeInclude}\n\n${funcContents}`, {
```

Comment:
I don't think we should reformat the user's code, that seems fairly invasive, and also outside the scope of the PR's intention.